crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -817,6 +817,8 @@ def do_controllers_substep(substep)
           upgrade_controller_clusters
           upgrade_non_compute_nodes
           prepare_all_compute_nodes
+          # Stop nova services on all nodes to prevent any and all RPC API trouble
+          stop_nova_services
           reload_nova_services
```

Comment:
this `reload_nova_services` call doesn't make much sense here now, does it?

Reply:
It starts the services. So it still makes sense, though the naming might be bit confusing

Reply:
I didn't change the name on purpose - didn't want to risk breaking something due to a dangling reference to the old name somewhere...

Reply:
ah, so the idea here is that we stop everything and then start it all again without anything in between, correct? if so then I missed the point before.

Reply:
Yes.